app/helpers/comments_helper.rb

```diff
@@ -23,16 +23,19 @@ def commentable_author_is_op?(commentable, comment)
   end
 
   def get_ama_or_op_banner(commentable)
-    commentable.decorate.cached_tag_list_array.include?("ama") ? "Ask Me Anything" : "Author"
+    if commentable.decorate.cached_tag_list_array.include?(I18n.t("helpers.comments_helper.ama"))
+      I18n.t("helpers.comments_helper.ask_me_anything")
+    else
+      I18n.t("helpers.comments_helper.author")
+    end
   end
 
   def tree_for(comment, sub_comments, commentable)
     nested_comments(tree: { comment => sub_comments }, commentable: commentable, is_view_root: true)
   end
 
   def should_be_hidden?(comment, root_comment)
-    # when opened by a permalink + root comment is hidden => show root comment and its descendants
-    comment.hidden_by_commentable_user && comment != root_comment && !root_comment&.hidden_by_commentable_user
+    comment.hidden_by_commentable_user && comment != root_comment
```

Comment:
This lost one condition, was this on purpose?

Reply:
Oops, thank you!